subjects/printevenarguments/README.md

```diff
@@ -0,0 +1,15 @@
+## printevenarguments
+
+### Instructions
+
+Write a program that receives some arguments from the command line and prints the ones with an even index followed by a new line. If the number of even arguments is 0, print a new line.
```

Comment:
what if the arg is 1, not even but not 0 as well?
```
$ go run . a | cat -e
```

Reply:
I guess that's in the description already? That would mean the number of even arguments is 0. I'll add one example of this to the readme. Thanks for your review.